modules/retail/pages/retail-requirements.adoc

```diff
@@ -70,7 +70,23 @@ xref:installation:general-requirements.adoc[].
 
 
 
-== POS Terminal UEFI Secure Booting Requirements
+== PoS Terminal Requirements
```

Comment:
Why 'PoS'?
I grepped docs for PoS and indeed we use it as well, however we use POS more often and is also used generally.